app/services/date_parser.rb

```diff
@@ -0,0 +1,13 @@
+module DateParser
+  AMERICAN_REGEX = %r{(?<month>\d{1,2})/(?<day>\d{1,2})/(?<year>\d{4})}.freeze
+
+  # Date parsing with a fallback for american-style Month/Day/Year
+  # since we have legacy data in PII bundles that may be stored this way
+  def self.parse(str, allow_american: true)
```

Comment:
Does the `allow_american` option provide much value if it's not something we use? I assume in cases where we might _want to_ to use it, we'd opt for `Date.parse` instead?

It leads into a secondary question about whether we'd want to name the class in a way that more clearly suggests its purpose to normalize _possibly-American-formatted_ dates. Thinking something like `NormalizedAmericanDate.parse`, etc.

Reply:
Yeah it was mostly a way for me to label the behavior, I was trying to keep the class name short... WDYT of `DateParser.parse_legacy`?

Reply:
> Yeah it was mostly a way for me to label the behavior, I was trying to keep the class name short... WDYT of `DateParser.parse_legacy`?

Much better 👍

Reply:
fixed in ff7eb94448f6f9b79d62198149c041953ab2d5d3